src/actions/breakpoints/tests/__snapshots__/syncing.js.snap

```diff
@@ -18,11 +20,34 @@ Object {
       "sourceUrl": "http://localhost:8000/gen.js",
     },
     "hidden": false,
+    "id": Object {
```

Comment:
why are storing the ID?

Reply:
this was an issue earlier on, when we were not syncing the id -- the result was that synced breakpoints could not be deleted/turned into conditionals.

Reply:
we need to store the id in the sync step because earlier we were not, and this was creating breakpoints that could not be changed/deleted. however it shouldnt be such a large object, just a string